examples/units/README.txt

```diff
@@ -1,5 +1,7 @@
 .. _units_examples:
 
+.. _units-examples-index:
```

Comment:
(ack NVM, I think I am wrong. These labels only get generated for the `.py` files converted to RST, not the README.txt files)